docs/src/components/FooterBackground/styles.module.css

```diff
@@ -0,0 +1,75 @@
+.footerBackground {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+
+  background: linear-gradient(
+      180deg,
+      var(--swm-white) 0%,
+      rgba(252, 252, 255, 0) 150.5%
+    ),
+    linear-gradient(
+      180deg,
+      var(--swm-blue-light-40) 35%,
+      var(--swm-blue-light-60) 100%
+    );
+  width: 100%;
+  height: 50vh;
+  overflow: hidden;
+
+  z-index: -100;
+}
+
+.container {
+  position: relative;
+  margin-top: 107px;
+}
+
+footer {
+  margin-top: -107px;
+}
+
+@media (max-width: 996px) {
+  .container {
+    margin-top: 122px;
+  }
+  footer {
+    margin-top: -122px;
+  }
+}
+
+@media (max-width: 700px) {
+  .container {
+    margin-top: 147px;
+  }
+  footer {
+    margin-top: -147px;
+  }
+}
+
+@media (max-width: 375px) {
+  .container {
+    margin-top: 173px;
+  }
+  footer {
+    margin-top: -173px;
+  }
+}
```

Comment:
I'm concerned why these values are so deviated 😄 Can we round them?

Reply:
They fix the gradient on the footer, so they are so specific to be perfect. Before it was connected to Footer component from Docusaurus so there was no need to do that, right now we need to use negative margin to move gradient down to cover footer on each media screen size

Reply:
I'm still against putting that much specific numbers, couple of pixels shouldn't break anything 😄 Let's try rounding it